internal/http/processor.go

```diff
@@ -205,12 +210,12 @@ func (p *processor) processSeasonPack() (int, error) {
 
 	clientCfg, ok := p.cfg.Config.Clients[clientName]
 	if !ok {
-		return domain.StatusClientNotFound, fmt.Errorf("client not found in config")
+		return domain.StatusClientNotFound, domain.StatusClientNotFound.Error()
```

Comment:
Isn't it a bit redundant to return both the status code and err with the same err? Might want to do return nil err here and return err where it's returned from some other call.